mne/viz/brain/tests/test_brain.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import numpy as np
+import os.path as path
+from mne import read_source_estimate
+from mne.datasets import sample
+from mne.viz import Brain, get_3d_backend
+from mne.viz.brain.view import TimeViewer, ColorBar
+from mne.viz.brain.colormap import _calculate_lut
+
+
+def test_brain_init(backends_3d):
+    """Test initialization of the Brain instance."""
+    backend_name = get_3d_backend()
+    data_path = sample.data_path()
+    hemi = 'both'
+    surf = 'inflated'
+    subject_id = 'sample'
+    subjects_dir = path.join(data_path, 'subjects')
+
+    pytest.raises(ValueError, Brain, subject_id=subject_id,
```

Comment:
use context manager. and if you try different values of hemi. just use parametrization